Main/Include/snowflake.h

```diff
@@ -1,3 +1,24 @@
+/*
+    Copyright (C) 2019 Joshua Wade, Henna Haahtti
+    Original file by https://github.com/Shenggan/, licensed under MIT
```

Comment:
There's no license header in the original file, but I'd suggest adding the contents of this file into this comment with "Original license below" or something:
https://github.com/Shenggan/SnowFlake/blob/master/LICENSE